app_dart/lib/src/service/scheduler.dart

```diff
@@ -244,13 +244,8 @@ class Scheduler {
   Future<void> triggerPresubmitTargets({
     required github.PullRequest pullRequest,
     String reason = 'Newer commit available',
+    RetryOptions retryOptions = const RetryOptions(maxAttempts: 3),
```

Comment:
Should this PR's description get update to retry ci.yaml script independently of the scheduling workflow.

Reply:
There's a few issues I've noticed that lead me to choose here:
1. https://github.com/flutter/flutter/issues/93923 - Failure in filtering the presubmit targets as the GitHub token failed to be auth
2. https://github.com/flutter/flutter/issues/87965 - Failed to find ci.yaml as force pushes require a bit of extra time
3. Network flakes where the ci.yaml contents were off by a few bytes
Currently we don't throw errors if the a build fails to schedule on LUCI, so this should be quick to retry. In the worst case, I expect it to only add 10 seconds to the runtime.